lib/manageiq/rpm_build/generate_gemset.rb

```diff
@@ -96,6 +98,44 @@ def scrub
         cleanse_gemset
       end
 
+      def generate_dependency_manifest
+        where_am_i
+
+        FileUtils.rm_rf(manifest_dir) if manifest_dir.exist?
+        FileUtils.mkdir(manifest_dir)
+
+        shell_cmd("gem install license_finder")
```

Comment:
Let's make this an explicit depedency in the Gemfile

Reply:
That's what I tried first, but license_finder runs in bundle context, and needs to be installed in the gem_path we override during build. I tried to manipulate the path at license_finder run time, but I thought it was easier to just install it here.

Reply:
That's weird...ok we'll let this go for now

Reply:
Chatted with @simaishi and it's more accurate to say that the license finder runs in a different "GEM_HOME" context:

https://github.com/ManageIQ/manageiq-rpm_build/blob/9ebebe3d4c780316a99e000e90bbc38cce29fed4/lib/manageiq/rpm_build/generate_gemset.rb#L32-L34

As such, the script can't access the license_finder bin script if it's installed in the Gemfile as part of the manageiq-rpm_build application.